harvester/create.go

```diff
@@ -42,6 +44,12 @@ func (d *Driver) Create() error {
 		CloudInitDisk(builder.CloudInitDiskName, builder.DiskBusVirtio, false, 0, *cloudInitSource).
 		EvictionStrategy(true).RunStrategy(kubevirtv1.RunStrategyRerunOnFailure)
 
+	if d.ClusterName != "" {
```

Comment:
could we also add those two information? thanks

```
	d.ClusterType = flags.String("harvester-cluster-type")
	d.ClusterID = flags.String("harvester-cluster-id")
```

Reply:
Hi @w13915984028, what are cluster type and id mean here? Are them related any field on Rancher? Thanks.

Reply:
I did not check the code too much. guess they represent some internal information for Rancher Manager about the guest cluster.
The idea is: if we pass `cluster-name`, then the existing `cluster-type`, `cluster-id` can also be passed to driver.

Reply:
Got it. Since Rancher v2.11.0 release date is close and this needs to update source code in Rancher, it's better to include this in next PR.